core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -3328,6 +3345,9 @@ private boolean syncMetadata(RpcContext rpcContext, LockedInodePath inodePath,
         try {
           UfsStatus[] children = ufs.listStatus(ufsUri.toString(), listOptions);
           if (children != null) {
+            // Since we know the number of elements, make sure we don't re-hash
+            // 0.75f is the default
+            statusCache = new HashMap<>(Math.round(children.length * 1.34f), 0.75f);
```

Comment:
Why do we need to keep the .75 load factor? Can we use a larger load factor?

Reply:
We can, but it could increase the time to look up items in the map. I didn't want to change the performance characteristics of the status cache

Reply:
From my understanding: higher load factor -> smaller array size -> more likely to have hash collisions -> longer lookup times